language/bytecode-verifier/src/abstract_state.rs

```diff
@@ -234,115 +234,115 @@ impl AbstractState {
             .is_empty()
     }
 
-    /// checks if `nonce` is freezable
-    pub fn is_nonce_freezable(&self, nonce: Nonce) -> bool {
-        let borrows = self.borrow_graph.all_borrows(nonce);
-        self.all_nonces_immutable(borrows)
+    /// checks if `id` is freezable
```

Comment:
same here and maybe everywhere, carrying id in the name seems more confusing to me. 
Again your call